hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/CassDefinitionsServiceImpl.java

```diff
@@ -1909,11 +1928,15 @@ private Dampening mapDampening(Row row) {
                     throw new IllegalArgumentException("Unexpected ConditionType: " + cond);
                 }
 
+                // TODO: ARE THESE TAGS STILL NEEDED BY THE HAWKULAR UI? THIS IS CLUNKY.
+                // I DON'T THINK THEY ARE USED. REMOVING...
```

Comment:
Today UI is fetching triggers/alerts mainly by triggerId, which is something composed with specific strings.
So, I'm not 100% sure but tags are not used in UI code.